packages/arb-token-bridge-ui/src/pages/api/screen.ts

```diff
@@ -2,11 +2,17 @@ import { NextApiRequest, NextApiResponse } from 'next'
 import { utils } from 'ethers'
 import withRetry from 'fetch-retry'
 
+const isProduction = process.env.NODE_ENV === 'production'
+
 function loadEnvironmentVariable(key: string): string {
   const value = process.env[key]
 
   if (typeof value === 'undefined') {
-    throw new Error(`Missing "${key}" environment variable`)
+    if (isProduction) {
+      throw new Error(`Missing "${key}" environment variable`)
```

Comment:
One clarification - This error will be thrown on server and not to the end user. They will get a plan 500 error response without this clarification. I hope that was the intended behaviour? Otherwise we need to properly construct the error response using `res.status(500).send(...)`

Reply:
<img width="992" alt="image" src="https://user-images.githubusercontent.com/7558499/221829528-971c000e-1586-4fd9-aa45-be22e976728f.png">
<img width="799" alt="image" src="https://user-images.githubusercontent.com/7558499/221829711-343cbfb5-3e12-4280-9342-1c682c7db404.png">

Reply:
Logic was changed, we now don't call the API anymore